python/smqtk/algorithms/relevancy_index/libsvm_hik.py

```diff
@@ -183,7 +183,7 @@ def get_vector(d):
             with open(self.descr_cache_fp, 'wb') as f:
                 cPickle.dump(self._descr_cache, f, -1)
 
-    def rank(self, pos, neg):
+    def rank(self, pos, neg, use_libsvm=False):
```

Comment:
This should be a constructor parameter instead of being specified here.

Reply:
Regardless of where this ends up, the new parameter needs to be documented in the method docstring.

Reply:
Thanks!